src/main/java/school/faang/user_service/exception/GlobalExceptionHandler.java

```diff
@@ -139,4 +139,20 @@ public ResponseEntity<String> handleRedisPublishingException(RedisPublishingExce
         log.error("RedisPublishingException: {}", e.getMessage(), e);
         return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(e.getMessage());
     }
-}
+
+    @ExceptionHandler(JsonProcessingException.class)
+    public ResponseEntity<ErrorResponse> handleJsonProcessingException(JsonProcessingException ex) {
+        log.error("Json processing exception: {}", ex.getMessage(), ex);
+        String errorMessage = extractMessage(ex.getMessage());
+        return ResponseEntity.status(HttpStatus.NOT_FOUND)
```

Comment:
Тут логически скорее ближе BAD_REQUEST нежели NOT_FOUND
```suggestion
        return ResponseEntity.status(HttpStatus.BAD_REQUEST)
```

Reply:
Подумали немножко и пришли к тому, что еще логичнее INTERNAL_SERVER_ERROR, поскольку эта ошибка может возникнуть на этапе сериализации SubscriptionEvent в json, то есть внутренняя ошибка на сервере, а не проблема некорректного запроса пользователя.